xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -126,7 +126,16 @@
         <summary>Determines whether the group collection contains a captured group identified by the specified name.</summary>
         <returns>
           <see langword="true" /> if the group collection contains a captured group identified by <paramref name="key" />; <see langword="false" /> otherwise.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Implementations can vary in how they determine the equality of objects; for example, the class that implements <xref:System.Collections.Generic.IReadOnlyDictionary%602> might use the <xref:System.Collections.Generic.Comparer%601.Default%2A?displayProperty=nameWithType> property, or it might implement the <xref:System.Collections.Generic.IComparer%601> method.   
```

Comment:
This one does not make sense to keep. It's explaining how the specific interface implementation of this API could vary.

Reply:
Agreed. Removed all instances of these